app/models/case/sar/offender.rb

```diff
@@ -56,6 +56,7 @@ def searchable_fields_and_ranks
         subject_aliases: "B",
         third_party_company_name: "B",
         third_party_name: "B",
+        probation_area: "B",
```

Comment:
Does this need to be made searchable?

Reply:
I've removed probation area from searchability.